src/input/input.directive.ts

```diff
@@ -11,7 +11,7 @@ import {
 @Directive({
     selector: "[igxInput]"
 })
-export class IgxInputClass implements DoCheck {
+export class IgxInputDirective implements DoCheck {
```

Comment:
This would again be inconsistent, as the `IgxLabel` is not called `IgxLabelDirective`. Please check all the directives, and if they all follow the `Igx<DirName>` format, then leave it as `IgxInput`. If not, then we should discuss which naming to go with.

Reply:
OK, I checked. The whole thing is totally inconsistent. The `IgxFilterDirvective`, `IgxRippleDirective`, `IgxDraggableDirective` and `IgxDroppableDirective` are all located under the `directives` folder and follow the `Igx<DirName>Directive` naming.

The `IgxLabel`, `IgxInputClass`, `IgxLayout` and `IgxButton` directives are all in their own folders and follow mostly the `Igx<DirName>` naming, except for the input. 

I think we should unify all this and either move them all under directives and call the classes uniformly, or move them all into their all folders and again name them uniformly, or move them all into their own folders under the `directives` folder and again name them all uniformly.